internal/mode/static/state/dataplane/types.go

```diff
@@ -68,10 +68,20 @@ type VirtualServer struct {
 	Hostname string
 	// PathRules is a collection of routing rules.
 	PathRules []PathRule
-	// IsDefault indicates whether the server is the default server.
-	IsDefault bool
+	// Additions is a list of config additions for the server.
+	Additions []*Addition
```

Comment:
I was thinking that it would save space since these additions can be shared across servers and matches, but since they are stored in a slice, I don't think it matters.